Assets/AppCenter/Editor/CreateManifest.cs

```diff
@@ -0,0 +1,148 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Diagnostics;
+using System.IO;
+using System.IO.Compression;
+using System.Linq;
+using System.Xml.Linq;
+#if UNITY_2018_1_OR_NEWER
+using UnityEditor.Build.Reporting;
+#endif
+using UnityEditor.Build;
+using UnityEditor;
+using UnityEngine;
+
+public class CreateManifest
+{
+    public static void ZipFile(string sourceFile, string destinationFile)
+    {
+        if (Application.platform == RuntimePlatform.WindowsEditor)
+        {
+
+        }
+        else if (Application.platform == RuntimePlatform.OSXEditor)
+        {
+            var process = new Process()
+            {
+                StartInfo = new ProcessStartInfo
+                {
+                    FileName = "/bin/bash",
+                    //Arguments = $"-c \"zip -r {destinationFile} {"appcenter-loader-release/*"} \"",
+                    Arguments = $"-c \"cd {"appcenter-loader-release"} ; zip -r {"../Assets/AppCenter/Plugins/Android/appcenter-loader-release.aar"} {"*"} \"",
+                    RedirectStandardOutput = true,
+                    UseShellExecute = false,
+                    CreateNoWindow = true,
+                }
+            };
+            process.Start();
+            process.WaitForExit();
+        }
+        else
```

Comment:
remove else block, seems not used

Reply:
Sorry forgot to remove the debug message...